src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -130,6 +130,39 @@ public void canExpireASingleOpenAwaitingPickupRequest() throws InterruptedExcept
     assertThat(response.containsKey("position"), is(false));
   }
 
+  @Test
+  public void canExpireRequestWhenTlrSettingsNotReadable() throws InterruptedException,
+    MalformedURLException, TimeoutException, ExecutionException {
+    
+    stubWithInvalidTlrSettings();
```

Comment:
We will probably need more tests for different responses from mod-configuration

Reply:
Again, this condition only triggers when the TLR object itself is unreadable, either because the settings are invalid or it's empty.